### PR TITLE
Adding 0.6 release announcement

### PR DESCRIPTION
**Pull request by gunnarmorling**
@jpechane Any thoughts?

### CODE REVIEW

File: blog/2017-09-21-debezium-0-6-0-released.adoc [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+== New connector features
+
+Now let's take a look at some of the changes around the specific Debezium connectors.
+The **MySQL connector** has seen multiple improvements, e.g.:
+
+* Snapshot consistency wasn't guaranteed before in some corner cases (https://issues.jboss.org/browse/DBZ-210[DBZ-210]); that's fixed now
+* DEC and FIXED types supported in the DDL parser (https://issues.jboss.org/browse/DBZ-359[DBZ-359]; thanks to https://github.com/ooooorz[Liu Hanlin]!)
+* UNION clause supported for ALTER TABLE (https://issues.jboss.org/browse/DBZ-346[DBZ-346])
+
+For the **MongoDB connector**, the way of serializing ids into the key payload of CDC events has changed (https://issues.jboss.org/browse/DBZ-306[DBZ-306]).
```
Comment (jpechane) [inline blog/2017-09-21-debezium-0-6-0-released.adoc]: It is not clear enough that this is a breaking change

File: blog/2017-09-21-debezium-0-6-0-released.adoc [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+* Views won't stop the connector any more (https://issues.jboss.org/browse/DBZ-319[DBZ-319])
+* Warnings and notifications emitted by the server are correctly forwarded to the log (https://issues.jboss.org/browse/DBZ-279[DBZ-279])
+
+Please refer to the http://debezium.io/docs/releases/#release-0-6-0[changelog] for an overview of all the 20 issues fixed in Debezium 0.6.0.
+
+== What's next?
+
+High on our agenda is exploring support for Oracle (https://issues.jboss.org/browse/DBZ-20[DBZ-20]).
+We are also looking into using another logical decoding plug-in (wal2json) for the Postgres connector, which would enable to use Debezium with Postgres instances running on Amazon RDS.
+Another feature being worked on by community member https://github.com/mtagle[Moira Tagle] is support for updates to the `table.whitelist` for existing connector instances.
```
Comment (jpechane) [inline blog/2017-09-21-debezium-0-6-0-released.adoc]: Maybe we can mention also MariaDB plans?

**Comment by gunnarmorling** [comment]
Yes, good idea.

**Comment by gunnarmorling** [comment]
@jpechane Updated as per your suggestions. Any other comments or can we publish it?

**Pull request merged**